specification/components/schemas/eRS-CancelReferral-Parameters.yaml

```diff
@@ -0,0 +1,75 @@
+title: eRS-CancelReferral-Parameters
+type: object
+required:
+  - resourceType
+  - meta
+  - parameter
+properties:
+  resourceType:
+    type: string
+    enum:
+      - 'Parameters'
+    example: 'Parameters'
+  meta:
+    type: object
+    required:
+      - profile
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CancelReferral-Parameters-1"
+          example: "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CancelReferral-Parameters-1"
+        minItems: 1
+        maxItems: 1
+  parameter:
+    type: array
+    description: |
+      | Field                             | Cardinality | Notes                                      |
+      | --------------------------------- | ----------- | ------------------------------------------ |
+      | `cancellationReason`                 | `1..1`      |  The reason why the referral is being cancelled |
+      | `cancellationComments`               | `0..1`      |  This may be mandatory if mandatory comment indicator is set to 'Yes' for the Cancellation Reason Type selected. Maximum 2000 characters. |
+    items:
+      anyOf:
+        - title: cancellationReason
+          description: Parameter to supply a cancellationReason
+          type: object
+          required:
+            - name
+            - valueCoding
+          properties:
+            name:
+              type: string
+              enum:
+                - cancellationReason
+              example: 'cancellationReason'
+            valueCoding:
+              type: object
+              required:
+                - system
+                - code
+              properties:
+                system:
+                  type: string
+                  example: __baseurl__/FHIR/STU3/CodeSystem/REFERRAL-CANCELLATION-REASON
+                code:
+                  type: string
+                  example: INTEND_PRIVATE
+        - title: cancellationComments
+          description: Parameter to supply the cancellationComments
+          type: object
+          required:
+            - name
+            - valueString
+          properties:
+            name:
+              type: string
+              enum:
+                - cancellationComments
+              example: 'cancellationComments'
+            valueString:
+              type: string
+              example: 'Patient intend to go private'
+
```

Comment:
For consistency with other endpoints, these should be extracted into separate files in the parameters directory

Reply:
I think parameters directory should only have a reusable to parameters such as service.yaml, ClinicType.yaml,..,etc. This is a functional specific parameter and be placed outside of parameter directory.